FirebaseInstallations/Source/Library/FIRInstallationsLogger.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import <FirebaseCore/FIRLogger.h>
+
+extern FIRLoggerService kFIRLoggerInstallations;
```

Comment:
@paulb777 I would like to double check if it it a correct place for this declaration.

Reply:
LGTM